Improve the docs for Avo (#6075)
segmentio/segment-docs: src/connections/destinations/catalog/avo/index.md

---

**Avo lets you find, fix, and prevent data quality issues upstream.** World class data and product teams at companies like Fender, IKEA, Wolt use Avo to guarantee event data quality upstream, so they can focus on building great user experiences. With Avo you get reliable data with less effort, by moving from reactive damage control to proactive data management and addressing your data quality issues at the source, where the data is created.

With [Avo](https://avo.app){:target="\_blank”} Inspector, data quality is no longer a dream, it’s a workflow.
[Inspector](https://www.avo.app/data-observability){:target="\_blank”} lets you find, triage, fix, and prevent data quality issues in your event based data. Launch Inspector to discover all your data quality issues and systematically work towards better data, one resolved issue at a time.

The Avo Inspector destination automatically extracts event schemas from your product events, sending only the signatures from the connected Segment sources to the Inspector API. **Avo Inspector receives no PII data from your source**.

{% include content/plan-grid.md name="actions" %}

## Supported methods

### Track events

> The Avo destination supports `Track` events.

Reviewer: When discussing Segment methods outside of code snippets, please capitalize them and avoid stylizing them like inline code.

```suggestion
The Avo destination supports Track events.
```

1. From the Segment web app, click **Catalog**, then click **Destinations**.
2. Find the Destinations Actions item in the left navigation, and click it.

> 3. Select [Avo](https://app.segment.com/goto-my-workspace/destinations/catalog/actions-avo) from the list of destinations, then click **Add destination**.

Reviewer: ```suggestion
3. Select [Avo](https://app.segment.com/goto-my-workspace/destinations/catalog/actions-avo){:target="\_blank”} from the list of destinations, then click **Add destination**.
```

5. Copy the API Key
   ![Copy API key](images/api-key.png)

### Configure Destination

#### Avo Inspector API Key

You can copy the API key from your source in Avo. The API key allows Avo to map the events from your Segment source to the Avo source, to accurately compare your source’s event schemas to your Tracking Plan in Avo.

#### Environment

Environment describes which app environment the source is sent from, `Development | Staging | Production`.

> Avo only generates issues for events in the `Production` environment, but you can see the event shapes for staging and development environments to make sure they are implemented correctly.

Reviewer: ```suggestion
Avo only generates issues for events in your `Production` environment, but you can see the event shapes for staging and development environments to make sure they are implemented correctly.
```

#### App Version Property

> App Version Property is an optional **(but recommended!)** field. Having accurate app release versions in Avo Inspector allows you to see how events change across releases. This will help you identify which releases an issue is impacting, and monitor for regressions in future releases after an issue has been resolved.

Reviewer: ```suggestion
App Version Property is an optional **(but recommended!)** field. Having accurate app release versions in Avo Inspector allows you to see how events change across releases. This helps you identify which releases an issue is impacting, and monitor for regressions in future releases after an issue has been resolved.
```
